### PR TITLE
B lorient553

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**
I wasn't able to provide more refs with the kind of descriptions that Wright gives.

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient553.xml [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+<locus from="36r"/>
+<title type="complete" xml:lang="en">Other prayers of the blessed Virgin Mary</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.10">
+<locus from="39v"/>
+<title type="complete" xml:lang="en">Absolution <foreign xml:lang="gez">አብስልዋጥስ፡</foreign>
+and other prayers e. g. of S. Matthew on <locus target="#f.42v">f. 42v</locus>.</title>
+<textLang mainLang="gez"></textLang>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient553.xml:109]: St, I think it should be a subunit

File: LondonBritishLibrary/orient/BLorient553.xml [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+<title type="complete" xml:lang="gez" ref="LIT1714Kestat">ክሥተት፡ ዘአርያም፡</title>
+<notatedMusic><desc>With musical notes.</desc></notatedMusic>
+<textLang mainLang="gez"></textLang>
+</msItem>
+</msItem>
+
+<msItem xml:id="ms_i1.14">
+<locus from="69r"/>
+<title type="complete" xml:lang="en" ref="LIT1931Mashaf#Blessings">Bededictions <foreign xml:lang="gez">ቡራኬ፡</foreign>, the first of which is ascribed
+to ʾabbā Sāmuʾel and the sixth to Basil</title>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient553.xml:141]: Benedictions

File: LondonBritishLibrary/orient/BLorient553.xml [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+    <desc type="OwnershipNote">The note states that the manuscript belongs to the church of <placeName ref="INS0101MadhaneAlam">Madḫāne ʿAlam</placeName>
+    </desc>
+    <q xml:lang="gez">ግንዘት፡ ዘቅዱስ፡ መድኃኔ፡ ዓለም።</q>
+
+</item>
+
+<item xml:id="e1">
+<locus target="#51r"/>
+<desc>A mention of the king ʾIyāsu.</desc>
+</item>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient553.xml:231]: Could you mark up his name as well?

File: LondonBritishLibrary/orient/BLorient553.xml [hunk 4]
```diff
@@ -0,0 +1,303 @@
...
+        <decoNote xml:id="b2" type="bindingMaterial">
+            <material key="wood"/>
+            <material key="leather"/>
+            <material key="cotton"/>
+              </decoNote>
+    </binding>
+</bindingDesc>
+
+</physDesc>
+<history>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient553.xml:258]: originDate could also be added based on the reign date

**Review by DariaElagina — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient553.xml (hunk 3 above)
Comment (MarcinKrawczuk) [reply LondonBritishLibrary/orient/BLorient553.xml:231]: If you tell me which Iyasu he means :D

File: LondonBritishLibrary/orient/BLorient553.xml [hunk 5]
```diff
@@ -0,0 +1,309 @@
...
+              </decoNote>
+    </binding>
+</bindingDesc>
+
+</physDesc>
+
+<history>
+  <origin>
+    <origDate evidence="internal" notBefore="1801" notAfter="1803">According to the notes on <locus target="#114v">f. 114v</locus> and <locus target="115r">115r</locus> the manuscript
+      was written in the reign of <persName ref="PRS3692egwalas">ʾƎgʷāla Ṣǝyon</persName> when the ʾabuna was <persName ref="PRS10450YosabI">Yosāb</persName>.</origDate>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient553.xml:263]: evidence="reign"

File: LondonBritishLibrary/orient/BLorient553.xml (hunk 3 above)
Comment (DariaElagina) [reply LondonBritishLibrary/orient/BLorient553.xml:232]: I think, Iyasu IV (PRS5619IyasuIV), all others are either too early or too late

**Review by DariaElagina — APPROVED** [review]

**Pull request merged**